epub33/rs/index.html

```diff
@@ -2188,15 +2186,26 @@ <h2>Change Log</h2>
 					>Working Group's issue tracker</a>.</p>
 
 			<section id="changes-latest">
-				<h3>Substantive Changes since the <a href="https://www.w3.org/TR/2021/WD-epub-rs-33-20210112/">First
-						Public Working Draft</a></h3>
+				<h3>Substantive Changes since the <a href="https://www.w3.org/TR/2021/WD-epub-rs-33-20210223/">Previous
+						Working Draft</a></h3>
 
 				<!--
 					After each working draft is published:
 						- change the link/text in the section heading to refer to the published draft (use the dated URL)
 						- move all changes down to the next section
 				-->
 
+				<ul>
+					<li>19-Feb-2021: Updated the scripting security recommendations so that the text discusses assigning
```

Comment:
Maybe "recommends" rather than "discusses"? Not sure of the ideal way to mention a non-normative change.

Reply:
That's okay. W3C specs include a reference to RFC 8174, which makes it formal that non-capitalized uses of 2119 keywords only have their plain English meaning. I've been meaning to give the docs a once over to undo some of the more egregious uses of "might" and "need"... :)